common/testutils/random/test_random.go

```diff
@@ -0,0 +1,60 @@
+package random
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+// TestRandom provides all the functionality of math/rand.Rand, plus additional randomness functionality useful for testing
+type TestRandom struct {
+	*rand.Rand
+}
+
+// NewTestRandom creates a new instance of TestRandom
+// This method may either be seeded, or not seeded. If no seed is provided, then current unix nano time is used.
+func NewTestRandom(fixedSeed ...int64) *TestRandom {
+	var seed int64
+	if len(fixedSeed) == 0 {
+		seed = time.Now().UnixNano()
+	} else if len(fixedSeed) == 1 {
+		seed = fixedSeed[0]
+	} else {
+		panic("too many arguments, expected exactly one seed")
+	}
+
+	fmt.Printf("Random seed: %d\n", seed)
+	return &TestRandom{
+		rand.New(rand.NewSource(seed)),
+	}
+}
+
+// GetRand returns the underlying random instance
+func (r *TestRandom) GetRand() *rand.Rand {
+	return r.Rand
+}
```

Comment:
nit: getter is not needed with a public field

Reply:
[617c2e3](https://github.com/Layr-Labs/eigenda/pull/976/commits/617c2e3b06115f7848a6288df411fbe6b9f55d37)